### PR TITLE
Zeto concept docs

### PR DESCRIPTION
**Pull request by jimthematrix**

### CODE REVIEW

File: doc-site/docs/architecture/zeto.md [hunk 1]
```diff
@@ -1,8 +1,122 @@
...
+- Token selector: when sending a transaction that transfers certain amount to another account, the transaction input is made up of a collection of UTXOs that will be spent. The UTXOs must be selected from the unspent UTXOs that the account holds. The selection process needs to take into account the intend transfer amount and the values of the available tokens.
+- ZK proof generator: each Zeto transaction must be accompanied by a ZK proof to demonstrate the validity of the transaction proposal. This is accomplished by a proof generator that is able to use the secrets known only to the Paladin runtime hosting the account key, as private input to the ZKP circuit.
+
+## Private ABI
+
+The private ABI of Zeto is implemented in [Go](https://github.com/LF-Decentralized-Trust-labs/paladin/tree/main/domains/zeto), and can be accessed by calling `ptx_sendTransaction` with `"type": "private"`.
+
```
Comment (awrichar) [inline doc-site/docs/architecture/zeto.md:23]: In addition to the methods here, there is also a `lockProof` method on the private Zeto ABI, which should be documented.

Comment (awrichar) [reply doc-site/docs/architecture/zeto.md:23]: Unless the "lockProof" on the private ABI is no longer needed, and we can simply use the public Zeto ABI when "lockProof" is needed. Just depends on if any Go logic is needed when locking.

**Comment by awrichar** [comment]
Could you also add a `README.md` in the `domains/zeto` folder to direct people to the docs page (similar to the one that exists for Noto and Pente)?

File: doc-site/docs/architecture/zeto.md [hunk 2]
```diff
@@ -1,8 +1,122 @@
...
+        },
+        {
+          "name": "amount",
+          "type": "uint256",
+          "internalType": "uint256"
+        }
+      ]
+    },
+    {
+      "name": "data",
```
[the remark this answers is not in the document]
Comment (jimthematrix) [reply doc-site/docs/architecture/zeto.md:69]: you are right, the contracts already do but the domain implementation still ignores it right now. will remove

[the remark this answers is not in the document]
Comment (awrichar) [reply doc-site/docs/architecture/zeto.md]: I think Zeto does use the contract's `data` param to pack in the Paladin transaction ID. I found with Noto that I also wanted to let the user add their own transaction ID or other info - so I actually took the user's `data` param from the private ABI _along with_ the Paladin transaction ID and any other domain stuff, and packed it all into the resulting `data` param on the contract.

So it likely isn't a 1:1 mapping - but I do think we'll want a `data` param on the private ABI.

**Review by awrichar — APPROVED** [review]

**Pull request merged**